public/styles/site/newshub.sass

```diff
@@ -32,31 +39,35 @@ $headerColour: #6990e0
 #sidebar_left 
   width: 250px
   height: 975px
-  margin-left: -258px
-  float: left
-  
+  left: -258px
+  top: 25px
+  position: absolute
 #sidebar_right 
   width: 250px
   height: 975px
-  float: right
-  margin-right: -258px
-  top: 125px
+  right: -258px
+  top: 25px
+  position: absolute
 #sidebar_middle
   width: 299px
   height: 602px
-  float: left
+  
 
 //columns
 .column1 
   max-width: 744px
-  float: left
+@media screen and (max-width: 760px) 
```

Comment:
Use `{` for blocks `}`

Reply:
this is scheduled for next release (complete reimplementation using proper sass syntax. html will change too)